build-scripts/build-lxc.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+lxc launch ubuntu:16.04 --ephemeral my-build
+lxc exec my-build -- snap install snapcraft --classic
+lxc exec my-build -- apt update
+lxc exec my-build -- git clone https://github.com/ubuntu/microk8s.git
+lxc exec my-build -- sh -c "cd microk8s && SNAPCRAFT_BUILD_ENVIRONMENT=host KUBE_VERSION=v1.15.7 snapcraft"
```

Comment:
I didn't know about the `--use-lxd` option. I pulled this from the contents of `docs/build.md` which I could probably just update rather than including this script.

Reply:
It appears that `snapcraft             3.8                         3440   stable    canonical✓    classic` does not include this option. I also tried `--use-lxc`.

Reply:
deleted this script